store/sqlstore/adapters.go

```diff
@@ -17,23 +17,18 @@ type jsonArray []string
 
 func (a jsonArray) Value() (driver.Value, error) {
 	var out bytes.Buffer
-	if err := out.WriteByte('['); err != nil {
-		return nil, err
-	}
+	_ = out.WriteByte('[')
```

Comment:
Hmm .. what is the reason for these? I can see that this won't return an error, but the code seemed idiomatic.

Reply:
It was a lot of lines that did nothing. Why not get rid of them for simplicity?

Reply:
It just makes to code simpler to read. Having `_` leads to the question "why are we ignoring the errors?". Then the user has to realize that it is being ignored deliberately because it is guaranteed to not return errors. I think the cost of this mental complexity is higher than saving a couple of lines of code.